### PR TITLE
feat: enable skipping e2e and docker pipelines

### PR DESCRIPTION
**Pull request by artemchernyak-outreach**
## What this PR does / why we need it

Allows skipping e2e and docker steps to save ci time and cost.

## Jira ID

[DT-2595]

## Notes for your reviewers

[DT-2595]: https://outreach-io.atlassian.net/browse/DT-2595?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: manifest.yaml [hunk 1]
```diff
@@ -16,6 +16,14 @@ arguments:
     description: Enables pre-releasing via semantic-release
     schema:
       type: boolean
+  ciOptions.skipE2e:
+    description: Disables the e2e ci pipeline
```
Comment (jkinkead) [inline manifest.yaml:20]: Capitalize here and below?
```suggestion
    description: Disables the e2e CI pipeline
```

File: manifest.yaml [hunk 2]
```diff
@@ -16,6 +16,14 @@ arguments:
...
+    schema:
+      type: boolean
```
Comment (jkinkead) [inline manifest.yaml:22]: I assume implicit default is `false` for stencil booleans?

**Review by jkinkead — APPROVED** [review]
LGTM (from what I understand of templates)!

File: manifest.yaml (hunk 2 above)
Comment (artemchernyak-outreach) [reply manifest.yaml:22]: Yes, it's just depending on the Go default value behavior.

**Pull request merged**